readme.txt

```diff
@@ -106,52 +106,53 @@ There are opportunities for developers at all levels to contribute. [Learn more
 = 4.6 =
 
 * Release date: February 8, 2017
-* Release post:
+* Release post: http://wp.me/p1moTy-40g
 
 **Exciting New Features and Improvements**
 
-* New Google Analytics module. #5603
+* Enable Google Analytics and insert the necessary tracking code with this new Jetpack feature.

```

Comment:
I like the notion of not having to touch a single line of code.  The module subheader reads `Set up Google Analytics without touching a line of code.` -- can we work that in here?